diff --git a/apps/aecore/src/aec_governance.erl b/apps/aecore/src/aec_governance.erl
index 49c6d24..01f9347 100644
--- a/apps/aecore/src/aec_governance.erl
+++ b/apps/aecore/src/aec_governance.erl
@@ -210,8 +210,8 @@ primop_base_gas(?PRIM_CALL_MAP_PUT                   ) -> 0;
 primop_base_gas(?PRIM_CALL_MAP_DELETE                ) -> 0;
 primop_base_gas(?PRIM_CALL_MAP_SIZE                  ) -> 0;
 primop_base_gas(?PRIM_CALL_MAP_TOLIST                ) -> 0;
-primop_base_gas(?PRIM_CALL_CRYPTO_VERIFY_SIG         ) -> 1300;    %% 700 for call + 1300 = 2000
-primop_base_gas(?PRIM_CALL_CRYPTO_VERIFY_SIG_SECP256K1 ) -> 1300;    %% 700 for call + 1300 = 2000
+%% primop_base_gas(?PRIM_CALL_CRYPTO_VERIFY_SIG         ) -> 1300;    %% 700 for call + 1300 = 2000
+%% primop_base_gas(?PRIM_CALL_CRYPTO_VERIFY_SIG_SECP256K1 ) -> 1300;    %% 700 for call + 1300 = 2000
 primop_base_gas(?PRIM_CALL_CRYPTO_ECVERIFY_SECP256K1 ) -> 1300;    %% 700 for call + 1300 = 2000
 primop_base_gas(?PRIM_CALL_CRYPTO_ECRECOVER_SECP256K1) -> 1300;    %% 700 for call + 1300 = 2000
 primop_base_gas(?PRIM_CALL_CRYPTO_SHA3               ) -> 30;   %% Same as gas cost for SHA3 instruction
